usage/camera/README.md

```diff
@@ -29,9 +29,9 @@ Advanced features, along with some hints and tips, are described in the followin
 libcamera is a new Linux API for interfacing to cameras. Raspberry Pi have been involved with the development of libcamera and are now using this sophisticated system for new camera software. This means Raspberry Pi are moving away from the firmware-based camera image processing pipeline (ISP) to a more open system.
 
 - [Libcamera's main website](http://libcamera.org/).
-- [Raspberry Pi libcamera implementation](../../linux/software/libcamera/README.md).
-- [Tuning guide for the Raspberry Pi cameras and libcamera](../../linux/software/libcamera/rpi_SOFT_libcamera_1p0.pdf)
-- [Writing a kernel module to support a new camera or capture chip](../../linux/software/libcamera/csi-2-usage.md)
+- [Raspberry Pi libcamera implementation](../../software/libcamera/README.md).
+- [Tuning guide for the Raspberry Pi cameras and libcamera](../../software/libcamera/rpi_SOFT_libcamera_1p0.pdf)
```

Comment:
I guess this should point at `..._1p2.pdf`

Reply:
Yes.

Reply:
Fixed.

Reply:
Lol, sorry for the conflict!

Reply:
Doh! 😆